pkgs/tools/inputmethods/footswitch/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, stdenv, fetchFromGitHub, pkg-config, hidapi }:
+
+stdenv.mkDerivation {
+  pname = "footswitch";
+  version = "1.0+20210317.aa0b10f";
+
+  meta = with lib; {
+    description = "Command line utlities for programming PCsensor and Scythe foot switches.";
+    homepage    = "https://github.com/rgerganov/footswitch";
+    license     = licenses.mit;
+    platforms   = platforms.linux;
+    maintainers = with maintainers; [ baloo ];
+  };
```

Comment:
I don’t know if there is an official guideline for that, but it’s conventional to have the meta information at the end of the file (cf. [nixpkgs-hammering](https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/attribute-ordering.md))